lib/logstash/inputs/kafka.rb

```diff
@@ -114,6 +114,11 @@ class LogStash::Inputs::Kafka < LogStash::Inputs::Base
   # `session.timeout.ms`, but typically should be set no higher than 1/3 of that value.
   # It can be adjusted even lower to control the expected time for normal rebalances.
   config :heartbeat_interval_ms, :validate => :number, :default => 3000 # Kafka default
+  # Controls how to read messages written transactionally. If set to read_committed, consumer.poll()
+  # will only return transactional messages which have been committed. If set to read_uncommitted'
+  # (the default), consumer.poll() will return all messages, even transactional messages which have
+  # been aborted. Non-transactional messages will be returned unconditionally in either mode.
+  config :isolation_level, :validate => :string, :default => "read_uncommitted" # Kafka default
```

Comment:
seems this can be only two values: `read_committed` and `read_uncommitted` 
... thus we should validate no other value is set

Reply:
Had missed that validation option. Applied just now (hopefully correctly).